precompiles/xcm/XCM.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.8.0;
+
+/**
+ * @title XCM interface.
+ */
+interface XCM {
+    /**
+     * @dev Withdraw assets using PalletXCM call.
+     * @return A boolean confirming whether the XCM message sent.
+     */
```

Comment:
It would be good to provide better documentation for the teams.

This will the main thing they'll be looking at most likely.

Reply:
Agree, let me fix it